doc/source/plugins/b111_execute_with_run_as_root_equals_true.rst

```diff
@@ -1,5 +1,11 @@
-------------------------------------------
-B111: execute_with_run_as_root_equals_true
-------------------------------------------
+--------------------------------------------------
+B111: Test for the use of rootwrap running as root
+--------------------------------------------------
 
-.. automodule:: bandit.plugins.exec_as_root
+This plugin has been deprecated.
```

Comment:
We should say removed, not deprecated.  Deprecated implies it still works and will be removed in the future.